src/Domain.LinnApps/PurchaseOrders/PurchaseOrderService.cs

```diff
@@ -1055,8 +1056,8 @@ private void PerformDetailCalculations(PurchaseOrderDetail current, PurchaseOrde
 
             current.DetailTotalCurrency = Math.Round(netTotal + current.VatTotalCurrency.Value, 2);
 
-            if (creating || (updated.OrderUnitPriceCurrency == current.OrderUnitPriceCurrency
-                && updated.OurUnitPriceCurrency != current.OurUnitPriceCurrency))
+            if ((creating || (updated.OrderUnitPriceCurrency == current.OrderUnitPriceCurrency
+                && updated.OurUnitPriceCurrency != current.OurUnitPriceCurrency)) && (documentType != "CO" && documentType != "RO"))
             {
                 // if order price hasn't been overridden but our price has changed, use conv factor
                 current.OrderUnitPriceCurrency = updated.OurUnitPriceCurrency * current.OrderConversionFactor;
```

Comment:
but then is it still possible to get into this 'else if' on the next line? If so that could allow current.OrderUnitPriceCurrency to still be updated? Like I said not too sure how this works so just an obseravtion